cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -159,6 +159,7 @@ func TestAddFlags(t *testing.T) {
 				Prefix:                "/registry",
 				CompactionInterval:    storagebackend.DefaultCompactInterval,
 				CountMetricPollPeriod: time.Minute,
+				DbMetricPollInterval:  storagebackend.DefaultDbMetricPollInterval,
```

Comment:
Nit: "DB"

Reply:
Do you prefer DBSizePollInterval or the current DBMetricPollInterval? I am using DBMetricPollInterval hoping that we can reuse the flag if we add new etcd metrics in future.

Reply:
If the goal is to compute multiple metrics, then the name is fine.